.github/workflows/backend.yml

```diff
@@ -33,7 +38,56 @@ jobs:
           go-version: '1.21.5' 
 
       - name: run tests
-        run: go test ./... 
+        run: go test -race -coverprofile=coverage.out -covermode=atomic ./...
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v4.0.1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          slug: incompetent-hosting-provider/monorepo
+
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+          cache: false
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v4
+        with:
+          # Require: The version of golangci-lint to use.
```

Comment:
Delete the comments please

(this applies to all of those template things)

Reply:
Comments removed for everything but tests.